app/controllers/users_controller.rb

```diff
@@ -6,15 +6,24 @@ def new
 
   def create
     @user = User.new(user_params)
+    @user.status = 'pending_email'
+    @user.role = 'member'
+
     if @user.save
-      redirect_to root_url, notice: "User was successfully created!"
+      send_confirmation_email(@user)
+
+      redirect_to root_url, notice: "User was successfully created! Please check your email and confirm account"
     else
       render :new
     end
   end
 
   private
 
+  def send_confirmation_email(user)
+    UserMailer.with(user: user).confirmation_email.deliver_later
+  end
+
   def user_params
```

Comment:
`permitted_user_params`?

Reply:
What do you mean mentioning `permitted_user_params`? You mean that `def user_params` should be replaced to `def permitted_user_params`? Is that that critical?